fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionKontaktOnskas.java

```diff
@@ -0,0 +1,70 @@
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.luae_na.v1.model.converter.certificate.question;
+
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.ANSWER_NOT_SELECTED;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.ANSWER_YES;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.KONTAKT_CATEGORY_ID;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.KONTAKT_ONSKAS_DELSVAR_ID_26;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.KONTAKT_ONSKAS_DELSVAR_TEXT;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.KONTAKT_ONSKAS_SVAR_BESKRIVNING;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.KONTAKT_ONSKAS_SVAR_ID_26;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.KONTAKT_ONSKAS_SVAR_JSON_ID_26;
+import static se.inera.intyg.common.luae_na.v1.model.converter.RespConstants.KONTAKT_ONSKAS_SVAR_TEXT;
+import static se.inera.intyg.common.support.facade.util.ValueToolkit.booleanValue;
+
+import se.inera.intyg.common.fkparent.model.converter.RespConstants;
+import se.inera.intyg.common.services.texts.CertificateTextProvider;
+import se.inera.intyg.common.support.facade.model.Certificate;
+import se.inera.intyg.common.support.facade.model.CertificateDataElement;
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigCheckboxBoolean;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueBoolean;
+
+public class QuestionKontaktOnskas {
+
+    public static CertificateDataElement toCertificate(Boolean kontaktOnskas, int index, CertificateTextProvider textProvider) {
+        return CertificateDataElement.builder()
+            .id(KONTAKT_ONSKAS_DELSVAR_ID_26)
+            .parent(KONTAKT_CATEGORY_ID)
+            .index(index)
+            .config(
+                CertificateDataConfigCheckboxBoolean.builder()
+                    .id(KONTAKT_ONSKAS_SVAR_ID_26)
+                    .text(textProvider.get(KONTAKT_ONSKAS_SVAR_TEXT))
+                    .description(textProvider.get(KONTAKT_ONSKAS_SVAR_BESKRIVNING))
+                    .label(textProvider.get(KONTAKT_ONSKAS_DELSVAR_TEXT))
+                    .selectedText(textProvider.get(ANSWER_YES))
```

Comment:
ANSWER_YES and ANSWER_NOT_SELECTED seems to be the actual texts and not TEXT_IDs. So do not use the textProvider but instead set them directly.

Reply:
Done, and updated tests in in configcheckboxbolean.